### ISSUE TITLE
Mapping of Properties to Netlify Nomenclature

### ISSUE THREAD

**Issue by jamesqquick** [sandrinodimattia/serverless-jwt#1]
The nomenclature in referencing access tokens in Netlify functions using Netlify identity is a bit different.

Netlify attaches a `clientContext` property to the Netlify Function `context` instead of an `identityContext`. Additionally, Netlify attaches a `user` property to the `clientContext` vs a `claims` object. 

![image](https://user-images.githubusercontent.com/5391915/90043430-b8fa4700-dc91-11ea-8bfd-01ff3a0f6a46.png)

Would it be worth updating the properties in the @serverless-jwt/netlify package to match? This way validation will occur in the same way regardless of where the tokens come from.

In summary...

- `identityContext` becomes `clientContext`
- `claims` becomes `user`
